Tests/NIOOpenSSLTests/OpenSSLIntegrationTest.swift

```diff
@@ -1245,4 +1245,42 @@ class OpenSSLIntegrationTest: XCTestCase {
 
         XCTAssertEqual(certificates.count, 1)
     }
+
+    func testRepeatedClosure() throws {
+        let serverChannel = EmbeddedChannel()
+        let clientChannel = EmbeddedChannel()
+
+        defer {
+            // We expect both casees to throw
+            _ = try? serverChannel.finish()
```

Comment:
`XCTAssertNoThrow(XCTAssertEqual(false, serverChannel.finish()))`?

Reply:
oops, should read comments :)

Reply:
then the other way around, `XCTAssertThrows`

Reply:
Addressed.